contentScript.js

```diff
@@ -359,97 +361,91 @@ const createForm = () => {
 
   // multiple members allow checkbox
   // let allowMultipleid = "allowMultiple";
-  // let allowMultipleInput = createInput(allowMultipleid, inputStyles, "checkbox", "");
+  // let allowMultipleInput = createInput(allowMultipleid, "", "checkbox", "");
   // allowMultipleInput.checked = allow_multiple;
   // let allowMultipleInputLabel = createLabel("multipleinputlabel", allowMultipleid, "Allow multiple members ", textLabelStyles);
   // let allowMultipleWarn = createWarningText("This will prevent automatic click on the Schedule Now button", warnLabelStyles);
 
   let continuousretryid = "continuousretry";
-  let continuousretryinput = createInput(continuousretryid, inputStyles, "checkbox", "");
+  let continuousretryinput = createInput(continuousretryid, "", "checkbox", "", 'form-check-input');
   continuousretryinput.checked = keeptryingcontinuously;
   let continuousretrylabel = createLabel("continuousretrylabel", continuousretryid, "Attempt to book continuosly ", textLabelStyles);
   let continuousretryWarn = createWarningText("This will keep looking for available slots on screen continously and automatically attempt to book a slot randomly, please check appointment details on the captcha page. FIRST AVAILABLE SLOT ON THE PAGE WILL BE SELECTED. Automatic captcha detection is supported only if this is checked/selected.", warnLabelStyles);
 
   let enableautorefreshid = "enableautorefresh";
-  let enableautorefreshinput = createInput(enableautorefreshid, inputStyles, "checkbox", "");
+  let enableautorefreshinput = createInput(enableautorefreshid, "", "checkbox", "", 'form-check-input');
   enableautorefreshinput.checked = enableAutoRefresh;
   let enableautorefreshlabel = createLabel("enableautorefreshlabel", enableautorefreshid, "Enable Auto Refresh on search by district ", textLabelStyles);
   let enableautorefreshWarn = createWarningText("Keep auto refreshing every 2 seconds in search with district. This work only if 'Attempt to book continuosly is selected' and search preference is set to District. Use this option carefully. Your access to cowin portal may be blocked if you send too many request during a short duration.", warnLabelStyles);
 
-
   let timeslotinputid = "timeslotinput";
-  let timeslotinput = createInput(timeslotinputid, inputStyles, "number", timeslotind);
+  let timeSlotSelector = createSelectInput(timeslotinputid, "", timeslotind)
+  let one = createSelectOptions("timeSlot-1", "Slot 1", 1, timeslotind == 1)
```

Comment:
You might as well write the timings for each slots

Reply:
I mean in the select text

Reply:
I thought of adding the timings but I am unsure if the slots are fixed. 
If you can confirm the slots are the following, I can add that onto the select fields:
1. 9:00 am to 11:00 am
2. 11:00 am to 1:00 pm
3. 1:00 pm to 3:00 pm
4.  3:00 pm to 5:00 pm

Reply:
Yeah, in most cases these are the slots. We have disclaimed that if these are not the slots, slot 2 will be selected by defualt.